index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Home</title>
+</head>
+
+<body>
+    <section class="header">
+        <div class="logo-Placeholder">
+            <p class="logo">My Logo</p>
+        </div>
+        <div class="menu"></div>
+
+    </section>
+    <section class="headline">
+        <h1>I’m Zahra <br> Glad to see you!</h1>
+        <h2>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h2>
```

Comment:
- [ ] The title `I’m a software developer!` is considered to be a sub-title. Because of this reason, it should be the only content inside the `h2` tag. 
- [ ] The rest of the content is a paragraph. In that case, I suggest you place them inside a `p` tag.

See guide:
```
<section class="headline">
   <h1> // your main title </h1>
   <h2> // your subtitle </h2>
   <p> // your content </p>
</section>
```